404 reported in verifyTokenWithoutName because test endpoint does not exist

Running the TCK against my product shows the following failure in the [org.eclipse.microprofile.jwt.tck.container.jaxrs.RequiredClaimsTest.verifyTokenWithoutName](https://github.com/eclipse/microprofile-jwt-auth/blob/f2a0557d1ec0544b9c404216ff847dc401814fb4/tck/src/test/java/org/eclipse/microprofile/jwt/tck/container/jaxrs/RequiredClaimsTest.java#L283) test:

```
expected [401] but found [404]

java.lang.AssertionError: expected [401] but found [404]
at org.testng.Assert.fail(Assert.java:94)
at org.testng.Assert.failNotEquals(Assert.java:496)
at org.testng.Assert.assertEquals(Assert.java:125)
at org.testng.Assert.assertEquals(Assert.java:372)
at org.testng.Assert.assertEquals(Assert.java:382)
at org.eclipse.microprofile.jwt.tck.container.jaxrs.RequiredClaimsTest.verifyTokenWithoutName(RequiredClaimsTest.java:291)
```

I believe this is a bug in the test application used by the TCK. The problem is that the endpoint class (https://github.com/eclipse/microprofile-jwt-auth/blob/f2a0557d1ec0544b9c404216ff847dc401814fb4/tck/src/test/java/org/eclipse/microprofile/jwt/tck/container/jaxrs/RequiredClaimsEndpoint.java) that defines the endpoints invoked by the tests doesn’t have a “/verifyTokenWithoutName” endpoint defined, while all of the other tests do.

--- a/tck/src/test/java/org/eclipse/microprofile/jwt/tck/container/jaxrs/RequiredClaimsEndpoint.java
+++ b/tck/src/test/java/org/eclipse/microprofile/jwt/tck/container/jaxrs/RequiredClaimsEndpoint.java
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2016-2017 Contributors to the Eclipse Foundation
+ * Copyright (c) 2016-2020 Contributors to the Eclipse Foundation
  *
  *  See the NOTICE file(s) distributed with this work for additional
  *  information regarding copyright ownership.
@@ -271,6 +271,12 @@ public class RequiredClaimsEndpoint {
     }
 
     @GET
+    @Path("/verifyTokenWithoutName")
+    public Response verifyTokenWithoutName() {
+        throw new RuntimeException("This method must not be invoked");
+    }
+
+    @GET
     @Path("/verifyTokenWithoutClaim")
     public Response verifyTokenWithoutClaim() {
         throw new RuntimeException("This method must not be invoked");
@@ -282,4 +288,3 @@ public class RequiredClaimsEndpoint {
         throw new RuntimeException("This method must not be invoked");
     }
 }
-

